x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/__mocks__/es_results.ts

```diff
@@ -4,29 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { SignalSourceHit, SignalSearchResponse, SignalAlertParams } from '../types';
+import { SignalSourceHit, SignalSearchResponse, AlertTypeParams } from '../types';
 
-export const sampleSignalAlertParams = (maxSignals: number | undefined): SignalAlertParams => ({
+export const sampleSignalAlertParams = (
+  maxSignals: number | undefined,
+  riskScore?: number | undefined
+): AlertTypeParams => ({
   ruleId: 'rule-1',
   description: 'Detecting root and admin users',
   falsePositives: [],
   immutable: false,
   index: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
-  interval: '5m',
-  name: 'Detect Root/Admin Users',
```

Comment:
I thought that the name and interval was required.

Reply:
@XavierM it's omitted https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/types.ts#L127

Reply:
Those two are omitted because they are part of the main base object of alerting and are no longer alert parameters. This fixes some typing issues and bugs we had around our TypeScript types to keep things accurate.